src/biogeochem/CNDriverMod.F90

```diff
@@ -108,7 +108,7 @@ subroutine CNDriverNoLeaching(bounds,
     ! stays synchronized with albedo calculations.
     !
     ! !USES:
-    use clm_varpar                        , only: nlevgrnd, nlevdecomp_full 
+    use clm_varpar                        , only: nlevdecomp_full
```

Comment:
I clean up when I see unused items like this.

Reply:
Thanks :-)